src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

```diff
@@ -406,17 +406,26 @@
 
     <xsd:complexType name="messenger">
         <xsd:sequence>
-            <xsd:element name="default-serializer" type="xsd:string" minOccurs="0" />
-            <xsd:element name="symfony-serializer" type="messenger_symfony_serializer" minOccurs="0" />
-            <xsd:element name="encoder" type="xsd:string" minOccurs="0" />
-            <xsd:element name="decoder" type="xsd:string" minOccurs="0" />
```

Comment:
why are the encoder and decoder gone ?

Reply:
I can't find them referenced anywhere - there is no `encoder` or `decoder` in the Configuration class. I think these were incorrectly *not* removed from some earlier change to messenger.